zap/src/main/dist/license/TheClarifiedArtisticLicense.htm

```diff
@@ -52,7 +52,7 @@
     manual page for each non-standard executable that clearly documents how it 
     differs from the Standard Version. 
     <LI>make other distribution arrangements with the Copyright Holder. 
-    <LI>permit and encourge anyone who receives a copy of the modified Package 
+    <LI>permit and encourage anyone who receives a copy of the modified Package 
```

Comment:
This may be a legal document, I can drop this (or any other file(s)), just let me know.

Reply:
Yes, better not change these.